includes/admin/helper/class-wc-helper-plugin-info.php

```diff
@@ -36,25 +36,27 @@ public static function plugins_api( $response, $action, $args ) {
 			return $response;
 		}
 
-		$found_plugin = null;
-
-		// Look through local Woo plugins by slugs.
-		foreach ( WC_Helper::get_local_woo_plugins() as $plugin ) {
-			if ( dirname( $plugin['_filename'] ) === $args->slug ) {
-				$plugin['_slug'] = $args->slug;
-				$found_plugin = $plugin;
-				break;
-			}
+		// Only for slugs that start with woo-
+		if ( 0 !== strpos( $args->slug, 'woo-' ) ) {
```

Comment:
If this unique enough? woo- seems generic, and wc- would be worse. Maybe something specific to ourselves like `woocommerce-com` which infers it's origin?

Reply:
It doesn't require uniqueness. If we come across a plugin with the slug "woo-something-else" which is not on WooCommerce.com, nothing really happens. But I did change it to what you suggested in a59ce3b6b13359ab74f6bfee877f5c10d3795529.